packages/dev-middleware/src/inspector-proxy/Device.js

```diff
@@ -411,6 +477,14 @@ export default class Device {
         });
       }
 
+      if (
+        this.#debuggerConnection?.customHandler?.handleDeviceMessage(
```

Comment:
There is this weird behavior from Flow that caused me to put it above the `if (this.#debuggerConnection != null)` check.

If I put it inside that function, the existing code (below `$FlowFixMe[unused-promise]`) breaks as Flow thinks `this.#debuggerConnection` can be `null` again (even if its wrapped inside that null-check function)

Reply:
✅ Applied this to the block